tests/test_modeling_tf_electra.py

```diff
@@ -16,6 +16,8 @@
 
 import unittest
 
+import tensorflow as tf
```

Comment:
This line should. be put inside the `is_tf_available()`

Reply:
Oh! Good catch :+1: should be fixed now.